### PR TITLE
Document directoryfile cli option

### LINKED ISSUE #562: Document persistent server lists PR
Related to: https://github.com/jamulussoftware/jamulus/pull/1867 (once it's merged).

In https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/wiki/en/Running-a-Server.md#command-line-options we need to document the new `--directoryfile` option for server list persistence.

Proposal: 
`--directoryfile` Enables directories to remember registered servers even if the directory is restarted. You'll need to specify a path to an existing and writable file in which server information is stored.

### PR DESCRIPTION
**Pull request by ann0see**
Fixes: https://github.com/jamulussoftware/jamuluswebsite/issues/562

# Does this need translation?

- [x] Yes 
- [ ] No 

# Changes

Documents the directoryfile option

### CODE REVIEW

File: wiki/en/Running-a-Server.md [hunk 1]
```diff
@@ -229,26 +229,27 @@ Most common functions in Jamulus can be set using the GUI, but these and others
...
- /Applications/Jamulus.app/Contents/MacOS/Jamulus --serverbindip 192.168.0.100 
+ /Applications/Jamulus.app/Contents/MacOS/Jamulus --serverbindip 192.168.0.100
 ```
 
 **Note**: Command-line options will not alter settings made in the GUI.
 
 - `-d`  or `--discononquit`   Disconnect all clients on quit. Normally, when a server is stopped or restarted, any clients that have not used their "Disconnect" buttons will re-establish connection when the server comes back up again. Using this option forces clients to manually re-establish their connections to the server.  
-- `-e`  or `--directoryserver`  Make the server public and sets its genre (see also `-o`). See [server types](#server-types) for further information.  
+- `-e`  or `--directoryserver`  Make the server public and sets its genre (see also `-o`). See [server types](#server-types) for further information.
+- `--directoryfile` Enable directory to remember registered servers even if the directory is restarted. You'll need to specify a path to an existing and writable file in which server information is stored.
```
Comment (ann0see) [inline wiki/en/Running-a-Server.md]: That's the important one...

**Review by ann0see — COMMENTED** [review]
See above...

**Comment by ann0see** [comment]
Not to be merged before the actual next release.

File: wiki/en/Running-a-Server.md (hunk 1 above)
Comment (pljones) [reply wiki/en/Running-a-Server.md]: > existing and

I fixed that bit :)  -- it creates the file if not present.

Comment (ann0see) [reply wiki/en/Running-a-Server.md]: Changed.

File: wiki/en/Running-a-Server.md [hunk 2]
```diff
@@ -229,26 +229,27 @@ Most common functions in Jamulus can be set using the GUI, but these and others
...
 ```shell
- /Applications/Jamulus.app/Contents/MacOS/Jamulus --serverbindip 192.168.0.100 
+ /Applications/Jamulus.app/Contents/MacOS/Jamulus --serverbindip 192.168.0.100
 ```
 
 **Note**: Command-line options will not alter settings made in the GUI.
 
 - `-d`  or `--discononquit`   Disconnect all clients on quit. Normally, when a server is stopped or restarted, any clients that have not used their "Disconnect" buttons will re-establish connection when the server comes back up again. Using this option forces clients to manually re-establish their connections to the server.  
-- `-e`  or `--directoryserver`  Make the server public and sets its genre (see also `-o`). See [server types](#server-types) for further information.  
+- `-e`  or `--directoryserver`  Make the server public and sets its genre (see also `-o`). See [server types](#server-types) for further information.
```
Comment (pljones) [inline wiki/en/Running-a-Server.md]: >  Make the server public and sets its genre

"Register the server with the specified directory".

Given that we support custom directories, "make public" and "genre" are out of date, really.

Comment (ann0see) [reply wiki/en/Running-a-Server.md]: That’s not related to this PR though. But OFC, we can change that.

**Pull request merged**